src/codegen.zig

```diff
@@ -171,6 +196,18 @@ pub fn generateSymbol(
                 ),
             };
         },
+        .Float => {
+            const float_bits = typed_value.ty.floatBits(target);
+            switch (float_bits) {
+                16 => writeFloat(f16, typed_value.val.toFloat(f16), target, endian, try code.addManyAsArray(2)),
+                32 => writeFloat(f32, typed_value.val.toFloat(f32), target, endian, try code.addManyAsArray(4)),
+                64 => writeFloat(f64, typed_value.val.toFloat(f64), target, endian, try code.addManyAsArray(8)),
+                80 => writeFloat(f80, typed_value.val.toFloat(f80), target, endian, try code.addManyAsArray(10)),
```

Comment:
Shouldn't the 10 match `f80`'s ABI size (12 on i386, 16 elsewhere)?

Reply:
`f80` is now explicitly unhandled in b0bb158. Happy to come back to this discussion when I start working on floats in x64. Until then, lowering floats was an unfortunate prerequisite to not regress comptime tests it seems.